wiki/Beatmap_Editor/Timing/en.md

```diff
@@ -1,41 +1,32 @@
+The timing for a [beatmap](/wiki/Beatmaps) consists of the song's offset and BPM (beats per minute).
+The song's offset is the amount of time between the start of an audio file and the first downbeat in the song.
+The song's BPM is essentially how fast a song is.
 
-The point of having correct timing is so that the timeline in the [editor](/wiki/Beatmap_Editor) lines up with the music. If the timing is incorrect, then the timeline does not match up with the song. Therefore, you cannot accurately place hit objects, and it becomes impossible to map correctly to the music.
+The primary reason to use correct timing is that the [timeline](/wiki/timeline) in the [editor](/wiki/editor) lines up with the music.
+If the timing is incorrect, the [hit objects](/wiki/hit objects) on the timeline will not snap to match up with the song.
+Therefore, you cannot accurately place hit objects and it will become nearly impossible to map correctly to the music.
 
-How to time songs
------------------
+---
 
-***Main page: [How to time songs](/wiki/Beatmap_Editor_Guides/How_to_time_songs)***
+For a guide on timing, see: [How to time songs](/wiki/How_to_time_songs)***
 
-Timing is extremely important. To time the song, you need to switch to timing tab.
-At the timing tab, there is a small box at the upper-right section, beside the timeline. This box will be used to set the approximate BPM based on your clicks.
+## Setup
 
-The mapper will play the song and listen to it. Press **«Tap Here»** in rhythm of the music.
-After a certain number of press, the approximate BPM will be set and the black keys will resonate as «tic toc toc toc» where tic is the start of new time signature (by 4/4 standard). Repeat at least five times and take the average reading. If you are not sure, ask for help in Chat Console or the forum. Upload your beatmap and use the link for request. You may have to do something in return.
+![Screenshot of the "Timing and Control Points" window](img/TimingSetup.jpg "The timing setup dialog shows the map's fluctuating BPM and sampleset changes")
 
-Again after the button is pressed enough, BPM sensitivity will change from 0,1 and 2.
+Timing sections allow you to vary the timing across the song.
+There are two types of timing sections: a regular timing section and an inherited timing section.
 
--   BPM Sensitivity 0: 999,00
--   BPM Sensitivity 1: 999,90
--   BPM Sensitivity 2: 999,99
-    -   *9 = random number / 0 = unaffected number.*
+A regular timing section is displayed as a red line in the editor.
+Every map has at least one.
+Some maps change their BPM during the song, however, and require multiple regular timing sections as a result.
 
-**The button «T» may also be used instead of «Tap Here!»**
+Inherited timing sections are displayed as a green line in the editor.
+These do not actually change the song's timing.
```

Comment:
try `They don't allow changes to a beatmap's timing, but focus on adjusting ...`